src/akamai/js/akamaiHD_flash.js

```diff
@@ -316,7 +316,25 @@ require("../../../html5-common/js/utils/constants.js");
       if (loaded && !rewind) return;
       else {
         try {
-          this.callToFlash("load("+rewind+")");
+
+          var originalAspectRatio = (9/16);
+
+          // JQuery height detect can fail if a height was not explicitly set.
+          // So we try to find one up the tree, and failing that we use a default common
+          // aspect ratio. Not ideal, but provides viewable result.
+          var wrapperWidth = $(videoItem).parent().parent().width();
+          var wrapperHeight = $(videoItem).parent().height();
```

Comment:
There was none other than an attempt to grab from a likely explicitly set location. But both work so I switched to just one parent step up.